src/ngraph/frontend/onnx_import/core/graph.cpp

```diff
@@ -75,7 +79,11 @@ namespace ngraph
             {
                 if (!m_model->is_operator_available(node_proto))
                 {
-                    unknown_operator_types.emplace(detail::to_string(node_proto));
+                    m_model->enable_opset_domain(detail::get_node_domain(node_proto));
```

Comment:
Can we separate this? The intention of the loop is to build the set of unsupported operators. This is a one-time operation, so having separate it is cleaner and more readable. Please don't forget to add an appropriate comment explaining what each of the loops does.

Reply:
Moreover if you would do this out of this loop you could iterate only over the `unknown_operator_types` list.